salttesting/jenkins.py

```diff
@@ -513,6 +516,8 @@ def get_minion_python_executable(options):
     try:
         python_executable = json.loads(stdout.strip())
         python_executable = python_executable[options.vm_name]
+        if options.test_with_python3:
+            python_executable = '/usr/bin/python3'
```

Comment:
You're forcing a path to the binary and this will not work with virtualenv's, however, for now, I don't think this will be a problem.